### DISCUSSION TITLE
Optical microscope

### DISCUSSION THREAD

**Discussion by GPana** [neuropoly/axondeepseg#383] [Show and tell]
Do you still have an optical microscope model, or you don't run that anymore?

Thx

**Comment by jcohenadad** [comment]
It is not in production, but we do have some optical microscope models "in the fridge". Could you post an example of an image you would like to segment, so we can check what would be the most appropriate model for your data?

**Comment by GPana** [comment]
Thx - please have a look and let me know

Images are not very good. I have been having trouble with my pathologist.

Let me know if an optical model of some kind would work


![DSL20004](https://user-images.githubusercontent.com/73719288/97912252-756d2180-1d44-11eb-8b9c-9469e4dd45fb.JPG)

![DSL20040](https://user-images.githubusercontent.com/73719288/97912264-7a31d580-1d44-11eb-90ef-90919d449447.JPG)

**Comment by jcohenadad** [comment]
@GPana the sample preparation (tearing) and acquisition (focus) makes it difficult to identify the axons on these images. Even with the naked eye, the segmentation of axon/myelin is difficult. 

Here is an example of a good quality optical image:

![Screen Shot 2020-11-02 at 3 08 22 PM](https://user-images.githubusercontent.com/2482071/97914158-72ab0600-1d1d-11eb-9e03-c9aa1e945467.png)

Acquisition details [here](https://journals.lww.com/prsgo/fulltext/2020/04001/abstract_61__neural_networks_for_nerve_analysis_.61.aspx)
